netket/hilbert/continuous_hilbert.py

```diff
@@ -0,0 +1,67 @@
+from typing import Tuple
+
+from .abstract_hilbert import AbstractHilbert
+
+class Particles(AbstractHilbert):
+    """Class for continuous Hilbert space.
+
+    This class definese the common interface that can be used to
+    interact with hilbert spaces in continuum.
+    """
+
+    def __init__(self, N: int, L: Tuple[float, ...], pbc: Tuple[bool, ...], ptype: str):
+        """
+        Initializes a continuous Hilbert space.
+
+        :param N: Number of particles
+        :param L: spatial extension in each spatial dimension
+        :param pbc: Whether or not to use periodic boundary conditions for each spatial dimension
+        :param type: Type of particles (bosonic,..)
```

Comment:
We don't use numpy style but google style
```suggestion
    Args:
        N: Number of particles
        L: spatial extension in each spatial dimension
        pbc: Whether or not to use periodic boundary conditions for each spatial dimension
        type: Type of particles (bosonic,..)
```

Reply:
Also, about the type of particles... @gcarleo what types do we expect? I think this is better to be a diffrent class (BosonicParticle, FermionicParticle...) than a string in here